api/app/models/spree/webhooks/subscriber.rb

```diff
@@ -34,14 +34,25 @@ def self.with_urls_for(event)
         )
       end
 
-      def self.supported_events(model)
-        result = default_events(model)
-        result += SUPPORTED_CUSTOM_EVENTS[model] if SUPPORTED_CUSTOM_EVENTS.include?(model)
-        result
-      end
+      class << self
+        def supported_events
+          Spree::Base.descendants.
+            select { |model| model.includes Spree::Webhooks::HasWebhooks }.
```

Comment:
we should use `included_modules.include?` I guess

Reply:
my bad